basic/03b_command_velocity.py

```diff
@@ -29,7 +29,7 @@
 freq    = freq_hz * 2.0 * pi  # [rad / sec]
 amp     = 1.0                 # [rad] (45 degrees)
 
-duration = 10.0               # [sec]
+duration = 4              # [sec]
```

Comment:
To avoid spamming this PR with duplicate comments, see previous comment corresponding to `10->4` seconds in the examples... Not a big deal, just wondering why it was done